packages/desktop-client/src/components/AppBackground.tsx

```diff
@@ -22,7 +22,7 @@ export function AppBackground({
   const transitions = useTransition(loadingText, {
     from: { opacity: 0, transform: 'translateY(-100px)' },
     enter: { opacity: 1, transform: 'translateY(0)' },
-    leave: { opacity: 0, transform: 'translateY(-100px)' },
+    leave: { opacity: 0, transform: 'translateY(100)' },
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏did you mean to change this to `100px`? Why do this?
```suggestion
    leave: { opacity: 0, transform: 'translateY(100px)' },
```

Reply:
Yes, it is smoother IMO, it prevents overlapping of the next loading message with the previous one.